packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -139,6 +139,26 @@ class FlutterDevice {
     return reports;
   }
 
+  // Lists program elements changed in the most recent reload that have not
+  // since executed.
+  Future<List<ProgramElement>> unusedChangesInLastReload() async {
+    final List<Future<List<ProgramElement>>> reports =
+        <Future<List<ProgramElement>>>[];
+    for (FlutterView view in views) {
+      reports.add(view.uiIsolate.getUnusedChangesInLastReload());
+    }
```

Comment:
Folks don't like curly braces around single statement blocks: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-braces-for-one-line-long-control-structure-statements